lib/models/user.rb

```diff
@@ -0,0 +1,50 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+require 'bcrypt'
+require_relative 'peep'
+
+
+class User
+  attr_reader :password
+  attr_accessor :password_confirmation
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :username, String
+  property :name, String
+  property :email, String
+  property :password_digest, Text
+
+  has n, :peep
+
+
+  validates_confirmation_of :password
+  validates_presence_of :email, :name, :username
+  validates_length_of :username, :within => 5..15
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)